spec/helpers/visit_helper_spec.rb

```diff
@@ -60,6 +60,14 @@ def visit
       expect(helper.prison_address).to start_with "1 Fort Road"
     end
 
+    it "provides illegal html in the address and doesn't return illegal html" do
+      allow(Rails.configuration.prison_data['Rochester']).
+        to receive(:[]).with('address') { ['Danger<script>ous', 'apoel'] }
+
+      expect(helper.prison_address).not_to match(/<script/)
+      expect(helper.prison_address).to match(/<br/)
```

Comment:
@threedaymonk D'you reckon it's worth having two tests for this (one for single line, one for multi-line)? My gut says not, but I'm open to suggestion…